CommonMC/src/SelectRecoMC_module.cc

```diff
@@ -103,7 +103,8 @@ namespace mu2e {
 	Comment("Min CaloShowerSim MC energy to include")};
       fhicl::Atom<double> CCME{ Name("CaloClusterMinE"),
 	Comment("Minimum energy CaloCluster to save (MeV)"), 10.0};
-   };
```

Comment:
Again this looks like an SU2020-specific hack.  Standard production should always have CRVCoincidenceCluster collections, though they may be empty.  This change doesn't have anything to do with RMC, so I'm confused why it's here.

Reply:
I've reverted to the original version of the SelectRecoMC module.